main.py

```diff
@@ -3645,7 +3645,7 @@ async def open_pack(interaction: discord.Interaction):
     await message.response.send_message(embed=embed, view=gen_view(user))
 
 
-@bot.tree.command(description="who thought this was a good idea")
+@bot.tree.command(description="who thought the cattlepass was a good idea")
```

Comment:
sounds a bit weird, maybe something like `who thought a cattlepass would be a good idea`